client/login/wp-login/index.jsx

```diff
@@ -138,6 +140,8 @@ class Login extends React.Component {
 		return (
 			<Main className="wp-login">
 				<PageViewTracker path="/login" title="Login" />
+				<GlobalNotices id="notices" notices={ notices.list } />
```

Comment:
It probably should land in the logged out layout component, but let's keep it here for the time being.